model/team_test.go

```diff
@@ -112,24 +104,20 @@ var tReservedDomains = []struct {
 
 func TestReservedTeamName(t *testing.T) {
 	for _, v := range tReservedDomains {
-		if IsReservedTeamName(v.value) != v.expected {
-			t.Errorf("expect %v as %v", v.value, v.expected)
-		}
+		actual := IsReservedTeamName(v.value)
+		assert.Equal(t, actual, v.expected)
```

Comment:
same comment as above, needs to be flipped.
https://godoc.org/github.com/stretchr/testify/assert#Equal

Reply:
fixed.